pkg/conversion/cloner.go

```diff
@@ -153,6 +153,36 @@ func (c *Cloner) deepCopy(src reflect.Value) (reflect.Value, error) {
 		err := fv(src.Interface(), outValue.Interface(), c)
 		return outValue, err
 	}
+
+	// Check for custom deep copy funcs on the pointer of this type
+	if src.Kind() == reflect.Struct {
```

Comment:
a reflect.Value is passed here... we hit this all over the place traversing struct fields (deepCopyDefault calls back here recursively)